code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -19,15 +19,14 @@
 	mob_name = "a swarmer"
 	death = FALSE
 	roundstart = FALSE
-	flavour_text = {"
-	<b>You are a swarmer, a weapon of a long dead civilization. Until further orders from your original masters are received, you must continue to consume and replicate.</b>
-	<b>Clicking on any object will try to consume it, either deconstructing it into its components, destroying it, or integrating any materials it has into you if successful.</b>
-	<b>Ctrl-Clicking on a mob will attempt to remove it from the area and place it in a safe environment for storage.</b>
-	<b>Objectives:</b>
+	important_info = "Follow your objectives, do not make the station inhospitable or try and kill crew."
+	//TODO line breaks aren't carried through to the description, meaning the objectives are all just in text rather than on their own lines. Need to figur eout how to fix that.
+	flavour_text = "You are a swarmer, a weapon of a long dead civilization. Until further orders from your original masters are received, you must continue to consume and replicate."
+	description = {" Your goal is to create more of yourself by consuming the station. Clicking on any object will try to consume it, either deconstructing it into its components, destroying it, or integrating any materials it has into you if successful. Ctrl-Clicking on a mob will attempt to remove it from the area and place it in a safe environment for storage.
```

Comment:
If line breaks arent supported, why are some still in here

Reply:
Comment is old, I figured out how to get linebreaks to work. Will fix shortly